config/e2e/monitoring.yaml

```diff
@@ -174,7 +174,7 @@ spec:
       meta:
         package:
           name: system
-          version: 8.4.0
+          version: 8.4.1
```

Comment:
I think this one is wrong since a while. The latest version of the system package is [`1.19.4`](https://github.com/elastic/integrations/blob/main/packages/system/changelog.yml). But I'd rather handle that in a separate PR.